osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -100,10 +100,12 @@ protected override void LoadComplete()
 
         public IEnumerable<KeyBinding> SongSelectKeyBindings => new[]
         {
+            new KeyBinding(InputKey.Left, GlobalAction.SelectPreviousBeatmap),
+            new KeyBinding(InputKey.Right, GlobalAction.SelectNextBeatmap),
```

Comment:
It's really confusing to a user to find these bindings, as the difficulty/beatmap level previous/next are in completely different sections). I think the new bindings should be introduce as global "Next Group" / "Previous Group" and used in more places where we expect such traversal to work.

Of note, `VolumeMeter` should also be changed to use these bindings. The current `SelectPrevious` / `SelectNext` are actually incorrect and don't work as a result.

Reply:
> Of note, `VolumeMeter` should also be changed to use these bindings. The current `SelectPrevious` / `SelectNext` are actually incorrect and don't work as a result.

I've checked on stable and it seems this is correct? left/right would change the volume meter selection, and up/down would change the volume of the currently selected volume meter.

Reply:
I'm actually wildly confused about what is going on with the volume meter stuff. Not only `VolumeMeter` using `GlobalAction.Select{Previous,Next}Group` looks wrong, but `GlobalAction.{Decrease,Increase}Volume` and `GlobalAction.{Previous,Next}VolumeMeter` already exist, and by default use arrows combined with Alt.

As far as I can tell, to match stable, `VolumeMeter` should continue to use `Select{Previous,Next}`, and the group ones should be used by `VolumeOverlay`? Not sure what to do with the binding duplication, though.